swarm/pss/protocol.go

```diff
@@ -0,0 +1,201 @@
+package pss
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/p2p"
+	"github.com/ethereum/go-ethereum/p2p/protocols"
+	"github.com/ethereum/go-ethereum/rlp"
+	whisper "github.com/ethereum/go-ethereum/whisper/whisperv5"
+	"time"
+)
+
+// protocol specification of the pss capsule
+var pssSpec = &protocols.Spec{
+	Name:       "pss",
+	Version:    1,
+	MaxMsgSize: defaultMaxMsgSize,
+	Messages: []interface{}{
+		PssMsg{},
+	},
+}
+
+// Bridges pss send/receive with devp2p protocol send/receive
```

Comment:
We should follow the [Go documentation conventions](https://blog.golang.org/godoc-documenting-go-code) and aim to start comments with the thing being described, so here it would be:

```
PssReadWriter bridges pss send/receive...
```